src/main/java/uk/gov/justice/probation/courtcasematcher/messaging/HearingExtractor.java

```diff
@@ -40,13 +40,13 @@ public class CourtCaseExtractor {
     @Value("${feature.flags.pass-hearing-id-to-court-case-service:false}")
     final boolean passHearingIdToCourtCaseService;
 
-    CourtCase extractCourtCase(String payload, String messageId) {
+    Hearing extractHearing(String payload, String messageId) {
         try {
             SnsMessageContainer snsMessageContainer = snsMessageWrapperJsonParser.parseMessage(payload, SnsMessageContainer.class);
             log.debug("Extracted message ID {} from SNS message of type {}. Incoming message ID was {} ", snsMessageContainer.getMessageId(), snsMessageContainer.getMessageType(), messageId);
 
             return switch (snsMessageContainer.getMessageType()){
-                case LIBRA_COURT_CASE -> libraParser.parseMessage(snsMessageContainer.getMessage(), LibraCase.class).asDomain();
+                case LIBRA_COURT_CASE -> libraParser.parseMessage(snsMessageContainer.getMessage(), LibraHearing.class).asDomain();
```

Comment:
I think renaming this enum will affect the message we receive as we need to match the message type.

Reply:
Yeah that's fine, technically from Libra a hearing and a case are one and the same anyway so this isn't incorrect

Reply:
@jhevans Is it ok to rename this as well?
`MessageType#LIBRA_COURT_CASE`